resources/js/pages/NewAddress.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="home">
+    <form @submit.prevent="handleSubmit">
+      <input placeholder="Building" class="form-control" v-model="building" />
```

Comment:
create input-label component (containing input and label) and use it instead of input